main.py

```diff
@@ -1,10 +1,42 @@
+import json
 import init_django_orm  # noqa: F401
-
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as players:
+        players_data = json.load(players)
+    for person in players_data:
+        email_to_add = players_data[person]["email"]
+        bio_to_add = players_data[person]["bio"]
+        race_to_add = players_data[person]["race"]
+        guild_to_add = players_data[person]["guild"]
+        skill_to_add = players_data[person]["race"]["skills"]
```

Comment:
U can make here for person, info in players_data:
and then it will more comfortable to make variables